content/docs/start/experiments.md

```diff
@@ -13,268 +11,205 @@ track experiments for you so there's no need to commit each one to Git. This way
 your repo doesn't become polluted with all of them. You can discard experiments
 once they're no longer needed.
 
-> 📖 See [Experiment Management](/doc/user-guide/experiment-management) for more
-> information on DVC's approach.
-
-## Running experiments
-
 Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
 [compare the changes](/doc/start/metrics-parameters-plots). Let's further
 increase the number of features in the `featurize` stage to see how it compares.
-
-`dvc exp run` makes it easy to change <abbr>hyperparameters</abbr> and run a new
-experiment:
-
-```dvc
-$ dvc exp run --set-param featurize.max_features=3000
-```
-
 <details>
+## Installing and Configuring the Project
 
-### 💡 Expand to see what happens under the hood.
```

Comment:
FYI in our engine `<details>` always need to be followed by a heading (any level of `#`s) which will be the section's title. Otherwise the app will [fail to build](https://dashboard.heroku.com/apps/dvc-org-iesahin-new-gs--oepxvh/activity/builds/e67c1ea9-6e13-42d0-bac2-ab535feb2835).

Reply:
Hmmm and BTW I'm not sure which one caused that build failure since they all seem to have a header 🤷